app/src/main/java/ua/in/badparking/ui/fragments/LocationFragment.java

```diff
@@ -91,9 +90,10 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         Location location = lm.getLastKnownLocation(LocationManager.NETWORK_PROVIDER);
         geocoder = new Geocoder(getActivity(), Locale.getDefault());
 
-        if (location != null) {
+        if (location != null && currentLocation == null) {
```

Comment:
that does mean that if more precise location comes - we will not update a marker?